lib/Backends/Interpreter/Interpreter.cpp

```diff
@@ -91,7 +91,8 @@ bool Interpreter::isOpSupported(const NodeInfo &NI) const {
   case Kinded::Kind::ResizeNearestNodeKind:
     return NI.allInputsAndOutputsHaveSameElemKind(
         {ElemKind::FloatTy, ElemKind::Float16Ty, ElemKind::Int8QTy,
-         ElemKind::Int16QTy, ElemKind::Int32QTy});
+         ElemKind::Int16QTy, ElemKind::Int32QTy, ElemKind::Int32ITy,
```

Comment:
Is `Int16QTy` actually supported? Doesn't look like it? Also, looks like it was broken before since `FloatTy` is listed but wasn't actually supported?

Reply:
Since the implementation is templatized. It is supported

Reply:
Ah, I thought this was CPUBackend, nevermind 🙂